### PR TITLE
KAA-250: Decouple Avro UI forms generator, use external avro ui artifacts. Improve stylesheets usage according to GWT style.

### PR DESCRIPTION
**Pull request by ikulikov**

### CODE REVIEW

File: server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/schema/BaseSchemaViewImpl.java [hunk 1]
```diff
@@ -113,7 +114,7 @@ protected void initDetailsTable() {
             detailsTable.setWidget(6, 1, schemaFileUpload);
         }
         else {
-            schema = new KaaAdminSizedTextArea(524288);
+            schema = new SizedTextArea(524288);
```
Comment (ghost) [inline server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/schema/BaseSchemaViewImpl.java]: Is it a random number or a deliberate one?

File: server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/schema/BaseSchemaViewImpl.java [hunk 2]
```diff
@@ -113,7 +114,7 @@ protected void initDetailsTable() {
             detailsTable.setWidget(6, 1, schemaFileUpload);
         }
         else {
```
Comment (ghost) [inline server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/schema/BaseSchemaViewImpl.java]: Code style.

File: server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/struct/BaseStructView.java [hunk 3]
```diff
@@ -158,7 +159,7 @@ private void init() {
 
         bodyLabel = new Label(Utils.constants.body());
         detailsTable.setWidget(1, 0, bodyLabel);
-        body = new KaaAdminSizedTextArea(524288);
+        body = new SizedTextArea(524288);
```
Comment (ghost) [inline server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/struct/BaseStructView.java]: Is it a random number or a deliberate one?

File: server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/struct/BaseStructView.java [hunk 4]
```diff
@@ -280,7 +281,7 @@ public void setData(T struct) {
         else {
```
Comment (ghost) [inline server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/view/struct/BaseStructView.java]: Code style

**Comment by ashvayka** [comment]
LGTM

**Pull request merged**